templates/zerver/help/at-mention-a-team-member.md

```diff
@@ -17,6 +17,10 @@ email address or part of the username of the user you want to @-mention.
 When you begin typing an email address or username, autocomplete suggestions
 will appear.
 
+    !!! tip ""
+        You can use **@all** to mention everyone subscribed to the stream you
```

Comment:
We probably want to put this in backticks, i.e. `@**all**`, so that people' don't try `@all` and then have it not work.  We have `@**name**` as the syntax for user @-mentions elsewhere in this document anyways.